cirq/ops/moment_test.py

```diff
@@ -59,10 +59,6 @@ def test_equality():
     eq.add_equality_group(Moment([cirq.X(b)]))
     eq.add_equality_group(Moment([cirq.Y(a)]))
 
-    # Equality depends on order.
-    eq.add_equality_group(Moment([cirq.X(a), cirq.X(b)]))
```

Comment:
think the test should be modified to add_equality_groups with these two instead of deleting (test that the order doesn't matter).